internals/secrethub/inject.go

```diff
@@ -62,6 +62,7 @@ func (cmd *InjectCommand) Register(r Registerer) {
 	).Short('c').BoolVar(&cmd.useClipboard)
 	clause.Flag("file", "Write the injected template to a file instead of stdout.").StringVar(&cmd.file)
 	clause.Flag("file-mode", "Set filemode for the file if it does not yet exist. Defaults to 0600 (read and write for current user) and is ignored without the --file flag.").Default("0600").SetValue(&cmd.fileMode)
+	clause.Flag("var", "Set variables to be used in templates.").Short('v').StringMapVar(&cmd.templateVars)
```

Comment:
Small thing: I don't really like the word `set`. It is rarely clear what `set` does. Something like this is maybe more clear:

`Define the value for a template variable with VAR=VALUE, e.g. --var env=prod`

Reply:
Also, are variables case sensitive? If so, state it in the docs.

IMO, they should be case *insensitive*.